indexer/app/lib/periodic_indexer.rb

```diff
@@ -278,72 +296,127 @@ def run_index_round
 
       checkpoints = []
 
-      did_something = false
```

Comment:
ack good catch. 

So, if I recall...I had totally removed the did_something and stuck it back half-ass'edly when I saw what it was doing...

But...does it really matter to save a commit to solr? Worse case, solr gets a softCommit request that does nothing...maybe I just remove the did_something?